### DISCUSSION TITLE
Product Hunt Launch April 17th

### DISCUSSION THREAD

**Discussion by tgberkeley** [reflex-dev/reflex#2931] [Announcements]
Reflex is launching on product hunt ([https://producthunt.com/products/reflex-5](https://t.co/QW2mNKuTcX))  April 17th!

For those of you on Product Hunt, we’d love to hear your thoughts and feedback. Support us by clicking the link and selecting notify me to be reminded on our launch day.

**Comment by abcnow** [comment]
The best way to attract people to use Reflex is by creating video tutorials every week.. as important as coding.. in fact your code can be better than react but if people don't learn how to use it your code doesn't exit...

**Reply by Alek99** [reply]
Good point we are currently working with a YouTuber on a video series which explains in detail how to create a reflex full stack with auth and db. This should be out in mid April.

We are planning on working with more. If you have any recommendations or YouTubers you like let us know and we can try and schedule something with them.
